contrib/docker/docker-compose.yml

```diff
@@ -1,4 +1,8 @@
 # docker-compose build && docker-compose up
+# If you experience problems with the CKAN container not being
+# able to connect to the DB, then most likely the DB has not
+# started up quickly enough. Just do "docker-compose up ckan"
+# again to retry
```

Comment:
would a hack like https://github.com/datacats/datacats/blob/master/datacats/scripts/install_postgis.sh#L10-L15 be of any use to us?